images/deployer/Dockerfile

```diff
@@ -9,4 +9,11 @@
 #
 FROM openshift/origin
 
+RUN yum install -y nfs-utils sudo
+
+# Edit sudoers file
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
```

Comment:
I have no problem making a new image for this purpose, but it is more invasive than continuing to use the recycler's current host image.